braindecode/augmentation/transforms.py

```diff
@@ -0,0 +1,1000 @@
+
+    Suggested in [1]_
+
+    Parameters
+    ----------
+    probability : float | callable
+        Either a float between 0 and 1 defining the
+        uniform probability of applying the operation, or a `callable`
+        (modelling a non-uniform probability law for example) taking no input
+        and returning a boolean.
+    ordered_ch_names : list
+        List of strings representing the channels of the montage considered.
+        Has to be in standard 10-20 style. The order has to be consistent with
+        the order of channels in the input matrices that will be fed to the
+        transform. This channel will be used to compute approximate sensors
+        positions from a standard 10-20 montage.
+    magnitude : float | None, optional
+        Float between 0 and 1 encoding the `max_degree` parameter:
+        ```
+        max_degree = magnitude * mag_range[1] + (1 - magnitude) * mag_range[0]
+        ```
+        Defaults to None (ignored).
+    mag_range : tuple of two floats | None, optional
+        Range of possible values for `max_degree` settable using the magnitude
+        (see `magnitude`). If omitted, the range (0, 30 degrees) will be used.
+    axis : 'x' | 'y' | 'z', optional
+        Axis around which to rotate. Defaults to 'z'.
+    max_degree : float, optional
+        Maximum rotation. Rotation angles will be sampled between `-max_degree`
+        and `max_degree`. Defaults to 15 degrees.
+    spherical_splines : bool, optional
+        Whether to use spherical splines for the interpolation or not. When
+        `False`, standard scipy.interpolate.Rbf (with quadratic kernel) will be
+        used (as in the original paper). Defaults to True.
+    random_state: int | numpy.random.Generator, optional
+        Seed to be used to instantiate numpy random number generator instance.
+        Defaults to None.
+
+    References
+    ----------
+    .. [1] Krell, M. M., & Kim, S. K. (2017). Rotational data augmentation for
+       electroencephalographic data. In 2017 39th Annual International
+       Conference of the IEEE Engineering in Medicine and Biology Society
+       (EMBC) (pp. 471-474).
+    """
+
+    def __init__(
+        self,
+        probability,
+        ordered_ch_names,
+        magnitude=None,
+        mag_range=(0, 30),
+        max_degrees=15,
+        random_state=None
+    ):
+        sensors_positions_matrix = torch.as_tensor(
+            get_standard_10_20_positions(ordered_ch_names=ordered_ch_names)
+        )
+        super().__init__(
+            probability=probability,
+            sensors_positions_matrix=sensors_positions_matrix,
+            magnitude=magnitude,
+            mag_range=mag_range,
+            axis='z',
+            max_degrees=max_degrees,
+            random_state=random_state
+        )
+
+
+class RandomYRotation(RandomSensorsRotation):
+    """Interpolates EEG signals over sensors rotated around the Z axis
```

Comment:
A fast copy paste ;)
```suggestion
    """Interpolates EEG signals over sensors rotated around the Y axis
```